crates/rome_bin/Cargo.toml

```
@@ -0,0 +1,29 @@
+[package]
+name = "rome_bin"
+version = "0.1.0"
+edition = "2021"
```

```suggestion
edition = "2021"
authors = ["Rome Tools"]
license = "MIT"
repository = "https://github.com/rome/tools"
```

We should also add the `description` field. I leave it to you because you have a better context.